base/all/DESCRIPTION

```diff
@@ -4,8 +4,25 @@ Title: PEcAn functions used for ecological forecasts and
     reanalysis
 Version: 1.6.0
 Date: 2018-08-20
+Authors@R: c(person("Mike","Dietze"),
+             person("David","LeBauer"),
+             person("Xiaohui", "Feng"),
+             person("Dan"," Wang"),
+             person("Carl", "Davidson"),
+             person("Rob","Kooper"),
+             person("Shawn", "Serbin"),
+             person("Ankur", "Desai"),
+             person("James", "Simkins"),
+             person("Chris", "Black"),
+             person("Istem", "Fer"),
+             person("Hamze", "Dookohaki"),
+             person("Alexey", "Shiklomanov"),
+             person("Elizabeth" , "Cowdery"),
+             person("Ann", "Raiho"),
+             person("Tess", "McCabe"),
+             person("Tony", "Gardella"))
 Author: David LeBauer, Mike Dietze, Xiaohui Feng, Dan Wang,
-    Mike Dietze, Carl Davidson, Rob Kooper, Shawn Serbin
+        Carl Davidson, Rob Kooper, Shawn Serbin
```

Comment:
We definitely do not. They should definitely be mutually exclusive. I'm surprised nobody else spotted this in review.

Reply:
When deleting Maintainer fields, let's be sure to transfer them to appropriate Role entries: Here, `person("David", "LeBauer", role = c("aut", "cre"), email = "dlebauer@illinois.edu")`

Reply:
...Or, uh, maybe @dlebauer uses a newer address now? But that format, anyway.

Reply:
Yep definitely do not have to keep them. My point I brought up in text I wrote in the description of my PR was about me not being sure about what people are supposed to be credited for what because Maintainer  and Author names are definitely out of date and I don't know what email people would like to list if at all. I could message everyone package by package and get things, but I seems better to just have people add themselves as they fix a package.